browser/css/w2ui-1.5.rc1.css

```diff
@@ -481,6 +481,8 @@ input[type=checkbox].w2ui-toggle:focus {
 .w2ui-overlay table.w2ui-drop-menu td.menu-text {
   padding: 8px 10px 8px 5px;
   width: auto;
+  text-align: center;
+  font-size: 16px;
```

Comment:
Please these additional rules at the end of the `toolbar.css` file. This way we keep the original w2ui file clean and less prone to changes and use the later file to overwrite those rules.

Also maybe we could use CSS var for the font-size, you can add a new one (example `--toolbar-drop-menu-font-size`) in the top of `color-palette.css`